### PR TITLE
Create Hello World!

### PR DESCRIPTION
**Pull request by gaurigadkari**
@renweview suggested this could be a good first PR, just say hello in the Hello World file.

I think it would be a good small change that would ensure people have done the setup correctly
https://github.com/WomenWhoCode/WWCodeMobile/issues/99

### CODE REVIEW

**Review by renweview — APPROVED** [review]

**Comment by vuinguyen** [comment]
@gaurigadkari Would it be a good idea to put an example of what folks should write in the Hello World file? 
Example:
Vui,  @vuinguyen,  I love open source 

And the issue could be to add a line similar to the example. I'm sure you will give directions in the issue about how to edit the file as well, but I think an example line here would help!

File: coding-projects/android/Hello World! [hunk 1]
```diff
@@ -0,0 +1 @@
+Hello, this is Gauri! I like going on walks and trying new food.
```
Comment (vuinguyen) [inline coding-projects/android/Hello World!:1]: Adding this line is definitely better! May I suggest taking a look at this [README file, from an open source workshop that I taught](https://github.com/vuinguyen/practice-git-skills/blob/master/FavoriteIceCreamFlavor/FavoriteIceCreamFlavor.md)

Even though it's a Hello World! file, I still added:

- a brief instruction at the top of the file
- requested that contributors add their Github handle as part of their answer

The nice thing about having Github handles in the file, is it makes it easy to keep track of everyone that contributed to this file. So when it comes time to gathering applauds, it's all there in the file and you don't have to find all the merged PR's connected to this file / issue 😊

**Review by vuinguyen — COMMENTED** [review]
@gaurigadkari I added a couple more comments. Let me know what you think!

File: coding-projects/android/Hello World!
Comment (lsireneva) [inline coding-projects/android/Hello World!:1]: Hi, I am Liubov Sireneva. My github lsireneva.

**Review by vuinguyen — APPROVED** [review]
Yes, I think adding the Github handle to the line will make it easier to keep track of everyone that's contributing to this file 👍 

Will approve @gaurigadkari

**Pull request merged**